### PR TITLE
LPS-106851 portal-search-web: Add ADTs for site facet widget

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-106851

@jonmak08

### CODE REVIEW

**Comment by oliv-yu** [comment]
ci:test:search

**Comment by oliv-yu** [comment]
ci:test:sf

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/site/facet/view.jsp [hunk 1]
```diff
@@ -19,85 +19,125 @@
...
-			persistState="<%= true %>"
-		>
-			<liferay-ui:panel
-				collapsible="<%= true %>"
-				cssClass="search-facet"
-				id='<%= renderResponse.getNamespace() + "facetScopePanel" %>'
-				markupView="lexicon"
-				persistState="<%= true %>"
-				title="site"
+		<aui:form method="post" name="siteFacetFm">
```
Comment (jonmak08) [inline modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/site/facet/view.jsp]: We typically name our forms `name="fm"` unless it has to be specifically named something else.

Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/site/facet/view.jsp]: Good to know! I'll update the other search-web widgets too.

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/css/_search_facet.scss [hunk 2]
```diff
@@ -1,6 +1,11 @@
 .search-facet {
 	.custom-control-label,
 	.term-name {
+		&.label.label-lg {
+			font-size: 12px;
+			padding: 5px 8px;
+		}
+
```
Comment (thektan) [inline modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/css/_search_facet.scss]: Sort

Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/css/_search_facet.scss]: My bad! Added

**Comment by thektan** [comment]
ci:test:sf

**Comment by thektan** [comment]
ci:test:search

**Comment by thektan** [comment]
Running search tests again in case `fm` changes caused any test failures.

**Comment by thektan** [comment]
@oliv-yu The labels still looked a bit off and noticed the clay labels had a different line-height, so I added `line-height: 1`.

**Before**
Screenshot 2020-02-21 17 48 23

**After**
Screenshot 2020-02-21 17 48 11

**Comment by thektan** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/85127. [See changes here.](https://github.com/thektan/liferay-portal/compare/4a153ed0932e~1...4a153ed0932e)

**Comment by thektan** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/85190. [See changes here.](https://github.com/thektan/liferay-portal/compare/11a0a04dd3c2~1...11a0a04dd3c2)

**Pull request closed without merging**